keras/backend/tensorflow_backend.py

```diff
@@ -2527,6 +2528,8 @@ def _make_callable(self, feed_arrays, feed_symbols, symbol_vals, session):
         # Handle fetches.
         for x in self.outputs + self.fetches:
             callable_opts.fetch.append(x.name)
+        if self._merged_summaries is not None:
+            callable_opts.fetch.append(self._merged_summaries)
```

Comment:
This raises an error for me, but replacing it for `self._merged_summaries.name` fixes the issue.